--- options.h
+++ options.h
@@ -23,6 +23,6 @@
 #ifndef DSS_PRIV_FILENAME
-#define DSS_PRIV_FILENAME "/etc/dropbear/dropbear_dss_host_key"
+#define DSS_PRIV_FILENAME "/Acess/Conf/dropbear/host_key_dss"
 #endif
 #ifndef RSA_PRIV_FILENAME
-#define RSA_PRIV_FILENAME "/etc/dropbear/dropbear_rsa_host_key"
+#define RSA_PRIV_FILENAME "/Acess/Conf/dropbear/host_key_rsa"
 #endif
@@ -55,2 +55,2 @@
 /* Enable X11 Forwarding - server only */
-#define ENABLE_X11FWD
+//#define ENABLE_X11FWD
@@ -163,3 +163,3 @@
 #ifndef MOTD_FILENAME
-#define MOTD_FILENAME "/etc/motd"
+#define MOTD_FILENAME "/Acess/Conf/dropbear/motd"
 #endif
@@ -216,2 +216,2 @@
  * rsa/dss key generation */
-#define DROPBEAR_URANDOM_DEV "/dev/urandom"
+#define DROPBEAR_URANDOM_DEV "/Devices/frandom"
