--- gcc/config.gcc
+++ gcc/config.gcc
@@ -519,3 +519,10 @@
 # Common parts for widely ported systems.
 case ${target} in
+*-*-acess2*)
+  extra_parts="crt0.o crtbegin.o crtend.o"
+  gas=yes
+  gnu_ld=yes
+  default_use_cxa_atexit=yes
+  use_gcc_stdint=provide
+  ;;
 *-*-darwin*)

@@ -1192,2 +1196,7 @@
 	;;
+i[3-7]86-*-acess2*)
+	tm_file="${tm_file} i386/unix.h i386/att.h dbxelf.h elfos.h i386/i386elf.h acess2.h"
+	tmake_file="i386/t-i386elf t-svr4"
+	use_fixproto=yes
+	;;
 i[34567]86-*-elf*)
